docs/modules/operation/pages/deep-dive/admin/housekeeping/introduction.adoc

```diff
@@ -27,20 +27,17 @@ SELECT eventid FROM notifications WHERE eventid = events.eventid)
 AND eventtime < now() - interval '6 weeks';
 ----
 
-== After Upgrading
+== After upgrading
 
-[[delete-cache-folder]]
-=== Delete cache folder
-Run the Fix Karaf script to delete cached items so that your system is cleaned up.
-We recommend doing this after an upgrade.
-This script tries to fix Karaf configuration problems by pruning `opt/opennms/data` and restoring all Karaf-related configuration files to a pristine state.
+=== Troubleshooting
 
-NOTE: Back up `/opt/opennms/etc` before running the script.
+If your {page-component-title} instance fails to start after upgrading, you can run the Fix Karaf script to delete cached items.
+The script tries to fix configuration problems by pruning `/opt/opennms/data` and restoring all Karaf-related configuration files to a pristine state.
 
 [source, console]
-----
 /opt/opennms/bin/fix-karaf-setup.sh
-----
+
+IMPORTANT: The Fix Karaf script can be destructive to files that may contain user-specific configuration settings.
```

Comment:
```suggestion
WARNING: The Fix Karaf script can be destructive to files that may contain user-specific configuration settings.
```
I think this is a bit stronger than Important, and might draw the reader's attention better.